lib/impress.application.js

```diff
@@ -575,6 +586,94 @@ impress.application.mixin = (application) => {
     }
   };
 
+  if (impress.mode === 'test') {
+    application.loadTests = (callback) => {
+      const relPath = '/test/';
+      const absPath = api.path.join(application.dir, relPath);
+      api.fs.readdir(absPath, (err, files) => {
+        if (err) {
+          application.log.error(impress.CANT_READ_DIR + absPath);
+          callback(err);
```

Comment:
Are you sure that it's needed to pass `err` into `callback` here?
Its value is ignored afterward anyway since it's only checked for existence, and this only leads to `impress` loading process being stopped, which makes this step really inconsistent with the behavior of `impress` when it encounters errors in other steps during the loading process.